parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -354,18 +364,31 @@ pub mod pallet {
 				payload.len() < T::MaxMessagePayloadSize::get() as usize,
 				SubmitError::MessageTooLarge
 			);
+
+			let base_fee = T::WeightToFee::weight_to_fee(
+				&T::WeightInfo::do_process_message()
+					.saturating_add(T::WeightInfo::do_commit_one_message()),
+			);
```

Comment:
Use `WeightToFee` to estimate the `base_fee`, benchmark shows that `T::WeightInfo::do_commit_one_message()>T::WeightInfo::do_commit_messages()/T::MaxMessagesPerBlock::get()`, so we charge based upon the former.